addons/account/models/account_move.py

```diff
@@ -3113,7 +3113,7 @@ def _get_computed_taxes(self):
             tax_ids = self.account_id.tax_ids
 
         if self.company_id and tax_ids:
-            tax_ids = tax_ids.filtered(lambda tax: tax.company_id == self.company_id)
+            tax_ids = tax_ids.filtered(lambda tax: tax.company_id == self.move_id.company_id)
```

Comment:
Isn't self.company_id related to the move ? Also, is it the right version to do it? It feels like something that should be done in 13.0.